testsuite/standalone/src/test/java/org/jboss/as/test/integration/management/cli/ModuleTestCase.java

```diff
@@ -186,6 +186,20 @@ public void addRemoveModuleMetaInf() throws Exception {
         testModuleAndSlotExist(false, slot, getModulePath());
     }
 
+    @Test
+    public void addModuleWithWhitespacesInDependencies() throws Exception {
+        // should trim the surrounding whitespaces
+        try {
+            cli.sendLine("module add --name=" + MODULE_NAME
+                    + " --resources=" + jarFile.getAbsolutePath()
+                    + " --dependencies=[ org.jboss.logging ]"
```

Comment:
I've seen this used in https://issues.jboss.org/browse/JBEAP-6282. I guess it's useful to visually separate the list of deps from rest of command for easier reading.

I thought the [] brackets are used for value lists (as in ArgumentWithListValue) in which case the use seems justified?